spec/features/user_votes_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+feature "User Votes ", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.